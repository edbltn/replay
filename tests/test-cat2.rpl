println(123 ^ "abc" ^ 1.5);
