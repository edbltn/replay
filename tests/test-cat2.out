123abc1.5
